mypyc/lib-rt/CPy.h

```diff
@@ -930,7 +925,7 @@ static PyObject *CPyIter_Send(PyObject *iter, PyObject *val)
     // (This behavior is to match the PEP 380 spec for yield from.)
     _Py_IDENTIFIER(send);
     if (val == Py_None) {
-        return CPyIter_Next(iter);
+        return PyIter_Next(iter);
```

Comment:
I think this is wrong, but I don't think it is any more wrong than it was before so we'll deal with it later

Reply:
(Wrong since it doesn't set `StopIteration` which the caller probably expects)

Reply:
Merged #666 which refines this fix.